views/LabelsBelowMap.svelte

```diff
@@ -59,13 +59,13 @@
 
 {#if item.options.labelsBelowMap === true}
   <div
-    class="q-locator-map-labels {item.options.labelsBelowMapOneRow === true ? 'q-locator-map-labels--one-row' : ''}">
+    class="s-q-item__annotation-legend s-font-note {item.options.labelsBelowMapOneRow === true ? 'q-locator-map-labels--one-row' : ''}">
     {#each numberedLabels as label}
-      <div class="q-locator-map-labels__label s-font-note">
-        <div class="q-locator-map-labels__icon" style={labelIconStyle}>
+      <div class="s-q-item__annotation-legend__item">
+        <div class="s-q-item__annotation-legend__item__icon">
           {@html label.icon}
```

Comment:
I think it would be great include the svg inline same as in chart and choropleth tools. So it will look really be exactly the same. What do you think @fgervasi ?

Reply:
You're right. I thought I'll take the annotations from the graphic. The legend will build like the other tools when there is possible.